From e4af2125c62e29a38290ebb490542ca18b1ff120 Mon Sep 17 00:00:00 2001
From: "miha.kralj" <miha.kralj@outlook.com>
Date: Mon, 20 Jul 2026 03:56:43 +0000
Subject: [PATCH] temp: vyos-1x-030-vpp-arm64-resource-defaults.patch

---
 interface-definitions/vpp.xml.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/interface-definitions/vpp.xml.in b/interface-definitions/vpp.xml.in
index bf5e56d..ea41e72 100644
--- a/interface-definitions/vpp.xml.in
+++ b/interface-definitions/vpp.xml.in
@@ -229,7 +229,7 @@
                         <regex>(auto)</regex>
                       </constraint>
                     </properties>
-                    <defaultValue>auto</defaultValue>
+                    <defaultValue>16384</defaultValue>
                   </leafNode>
                   <leafNode name="data-size">
                     <properties>
@@ -303,7 +303,7 @@
                       <help>Main heap size</help>
                       #include <include/unformat_memory_size.xml.i>
                     </properties>
-                    <defaultValue>3G</defaultValue>
+                    <defaultValue>256M</defaultValue>
                   </leafNode>
                   <leafNode name="main-heap-page-size">
                     <properties>
-- 
2.39.5

