diff --git a/interface-definitions/interfaces_ethernet.xml.in b/interface-definitions/interfaces_ethernet.xml.in
index 48f409b..4107d05 100644
--- a/interface-definitions/interfaces_ethernet.xml.in
+++ b/interface-definitions/interfaces_ethernet.xml.in
@@ -107,12 +107,6 @@
                   <valueless/>
                 </properties>
               </leafNode>
-              <leafNode name="lro">
-                <properties>
-                  <help>Enable Large Receive Offload</help>
-                  <valueless/>
-                </properties>
-              </leafNode>
               <leafNode name="rps">
                 <properties>
                   <help>Enable Receive Packet Steering</help>
@@ -131,12 +125,6 @@
                   <valueless/>
                 </properties>
               </leafNode>
-              <leafNode name="tso">
-                <properties>
-                  <help>Enable TCP Segmentation Offloading</help>
-                  <valueless/>
-                </properties>
-              </leafNode>
             </children>
           </node>
           <leafNode name="speed">
