From 146d57fe6b900d20e702978f070865f462a17c93 Mon Sep 17 00:00:00 2001
From: "miha.kralj" <miha.kralj@outlook.com>
Date: Mon, 20 Jul 2026 03:56:43 +0000
Subject: [PATCH] temp: vyos-1x-014-download-chunk-size.patch

---
 python/vyos/remote.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/vyos/remote.py b/python/vyos/remote.py
index a3cbc1a..c93ced2 100644
--- a/python/vyos/remote.py
+++ b/python/vyos/remote.py
@@ -59,7 +59,7 @@
 from vyos.base import Warning
 from vyos.defaults import directories
 
-CHUNK_SIZE = 8192
+CHUNK_SIZE = 1048576
 
 class InteractivePolicy(MissingHostKeyPolicy):
     """
-- 
2.39.5

